homework/style.css

```diff
@@ -1,3 +1,137 @@
-.alert-error {
-  color: red;
+.alert-error { 
```

Comment:
On a mobile phone you are not using the full (scarce) screen width:

![image](https://user-images.githubusercontent.com/2788771/65776815-1ef5e280-e143-11e9-8996-b6d687eadd79.png)